app/src/bcwallet-theme/features/person-flow/screens/PersonCredentialLoading.tsx

```diff
@@ -170,6 +176,8 @@ const PersonCredentialLoading: React.FC<PersonProps> = ({ navigation }) => {
       if (!status) {
         setDidCompleteAttestationProofRequest(false)
         navigation.goBack()
+      } else {
+        setStep(4)
```

Comment:
moving step 4 here prevents the progress bar from moving to step 4 when authenticateWithServiceCard completed but by being cancelled